<?xml version="1.0"?>
<launch>
  <arg name="num_samples" default="1000" />
  <arg name="chain_start" default="panda_link0" />
  <arg name="chain_end" default="panda_link8" />
  <arg name="timeout" default="0.005" />

  <param name="robot_description" textfile="$(find franka_description)/robots/panda_arm_instrumrnt.urdf" />

  <node name="robot_inverse_single" pkg="franka_teleopreation" type="robot_inverse_single" output="screen">
    <param name="num_samples" value="$(arg num_samples)"/>
    <param name="chain_start" value="$(arg chain_start)"/>
    <param name="chain_end" value="$(arg chain_end)"/>
    <param name="timeout" value="$(arg timeout)"/>
    <param name="urdf_param" value="/robot_description"/>
  </node>


</launch>
